### PR TITLE
fix: add docs for session options

### PR DESCRIPTION
**Pull request by ivarconr**
## About the changes
Add documentation for configurable session options, including `clearSiteDataOnLogout` coming in v4.13.

### CODE REVIEW

File: website/docs/deploy/configuring-unleash.md [hunk 1]
```diff
@@ -254,57 +258,55 @@ If you want to read content from a file, you should either initialize Unleash vi
...
+};
 ```
 
 </TabItem>
 
 <TabItem value="env" label="Environment variables (bash)">
 
-``` bash title="Enable self-signed certificates"
-DATABASE_SSL="{ \"rejectUnauthorized\": false, \"key\": \"$(cat /path/to/server-certificates/root.crt)\" }"
+```bash title="Enable self-signed certifsessione, "key": "$(cat /path/to/server-certificates/root.crt)" }"
```
Comment (thomasheartman) [inline website/docs/deploy/configuring-unleash.md:308]: Whoops, I think something happened here. It looks you deleted part of the example but added a little bit extra at the same time? 😅

File: website/docs/deploy/configuring-unleash.md [hunk 2]
```diff
@@ -191,43 +198,40 @@ The logger interface with its `debug`, `info`, `warn` and `error` methods expect
 
 ## Database configuration
 
-:::info
-In-code configuration values take precedence over environment values: If you provide a database username both via environment variables and in code with the Unleash options object, Unleash will use the in-code username.
-:::
+:::info In-code configuration values take precedence over environment values: If you provide a database username both via environment variables and in code with the Unleash options object, Unleash will use the in-code username. :::
```
Comment (thomasheartman) [inline website/docs/deploy/configuring-unleash.md:201]: Putting this all on one line breaks the rendered formatting. The final ::: need to be on their own line. I suggest we revert the changes for this paragraph.

File: website/docs/deploy/configuring-unleash.md [hunk 3]
```diff
@@ -2,9 +2,8 @@
 id: configuring_unleash
 title: Configuring Unleash
 ---
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
 
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
```
Comment (thomasheartman) [inline website/docs/deploy/configuring-unleash.md:6]: While this doesn't break anything, I'd prefer it if the imports stayed on separate lines. Maybe that's a prettier setting?

File: website/docs/deploy/configuring-unleash.md (hunk 2 above)
Comment (ivarconr) [reply website/docs/deploy/configuring-unleash.md:201]: fixing

**Comment by ivarconr** [comment]
> It might not matter much in this example, though. Happy to hear your thoughts or for you just to merge it 

I should not matter. It only matters when we explain how to contribute to unleash, but that is separate from the general config docs.
